### PR TITLE
Appimage

### PR DESCRIPTION
**Pull request by core1024**
All needed assets and Makefile for building an AppImage

### CODE REVIEW

File: stremio.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Name=Stremio
+Comment=Video organizer for your Movies, TV Shows and TV Channels
+Exec=./stremio %U
+Icon=stremio
+Terminal=false
+Type=Application
+Categories=GNOME;GTK;AudioVideo;
```
Comment (bilelmoussaoui) [inline stremio.desktop]: The categories are incorrect. The application doesn't use Gtk nor it's an application that follows GNOME's HIG.

File: stremio.desktop [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Name=Stremio
+Comment=Video organizer for your Movies, TV Shows and TV Channels
+Exec=./stremio %U
```
Comment (bilelmoussaoui) [inline stremio.desktop]: No need to specify a relative path here. It should be `stremio %U`

File: stremio.desktop (hunk 1 above)
Comment (Ivshti) [reply stremio.desktop]: thanks for noticing this, it might've been an artifact from Electron times, but even then "Gnome" would've been wrong

Comment (bilelmoussaoui) [reply stremio.desktop]: @Ivshti Is there a way to contact devs about another packaging platform or to help make the Linux users' experience much better? 
Thanks!

Comment (Ivshti) [reply stremio.desktop]: just open an issue in stremio-shell

**Comment by core1024** [comment]
Closing this as it's no longer relevant.

**Pull request closed without merging**